server/athenian/api/openapi/openapi.yaml

```diff
@@ -838,6 +838,219 @@ paths:
 # {% endif %}
       x-openapi-router-controller: athenian.api.controllers.reposet_controller
       x-codegen-request-body-name: body
+  /team/create:
+    post:
+      operationId: create_team
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TeamCreateRequest'
+        x-body-name: body
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreatedIdentifier'
+          description: Identifier of the created team.
+        403:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The user is forbidden to create a team from the specified items.
+        409:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: There is an existing team with the same contents.
+      security:
+      - bearerAuth: []
+      summary: Create a team.
+      tags:
+      - team
+# {% if False %}
+      - default
+# {% endif %}
+      x-openapi-router-controller: athenian.api.controllers.team_controller
+      x-codegen-request-body-name: body
+  /team/{id}:
+    delete:
```

Comment:
We are likely to use ARRAY the same way as in reposets to migrate less tables, though IDK exactly how Marvin sees it.

Reply:
A problem I see if we embed the users in the teams is that they can be outdated easily if we do not implement other update processes.
Also, if at some point we need to change their schema, the migration process would be also more complex.

Reply:
Yup, I'd rather user an ARRAY as we do for `reposets` 👍

Reply:
are we embedding the whole user or only its id?

Reply:
Only the ids - or, more precisely, their logins, if we allow out-of-account members

Reply:
then lgtm.
(`login` can change, so it might be not the best way to identify the users in the embedded list)

Reply:
I still need to better see how this could be done, do you need the answer now? I don't think that the way it is going to implemented will change the spec.

Reply:
I don't need an answer. I'm just providing info :D